api/mraa/common.h

```diff
@@ -234,6 +234,42 @@ unsigned int mraa_get_platform_pin_count(uint8_t platform_offset);
 char* mraa_get_pin_name(int pin);
 
 /**
+* Get pin number, board must be initialised.
+*
+* @param pin_name: GPIO Pin Name. Eg: IO0
+* @return int of MRAA index for gpio
```

Comment:
Is there a not found case?

Perhaps:
@return GPIO index for pin_name or -1 if not found

Applies to all mraa_*_lookup methods